src/Root.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {render} from '@testing-library/react';
+import {GlobalStyles} from './global';
+import {ThemeProvider} from "styled-components";
```

Comment:
you can use jest-styled-components lib for checking styling in better detail, even using snapshots, but good for now